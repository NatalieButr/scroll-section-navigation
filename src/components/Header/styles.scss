header {
    height: 60px;
    width: 100%;
    background: white;
    display: flex;
    align-items: center;
    padding-left: 20px;
    position: fixed;
}