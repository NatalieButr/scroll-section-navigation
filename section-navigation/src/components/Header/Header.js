import React, {useState} from 'react';
// import {Link} 

import './styles.scss'


function Header()  {
    const [links] = useState(['Meню', 'O нас', 'Контакты']);


    return (
        <header>
            <div className='header_logo'>
                <p className='text header_logo__text'>Harry Potter Section Navigation</p>
            </div>
        </header>
    )
    
}

export default Header;