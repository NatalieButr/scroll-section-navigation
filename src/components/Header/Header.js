import React from 'react';

import './styles.scss'


function Header()  {

    return (
        <header>
            <div className='header_logo'>
                <p className='text header_logo__text'>Harry Potter Section Navigation</p>
            </div>
        </header>
    )
}

export default Header;