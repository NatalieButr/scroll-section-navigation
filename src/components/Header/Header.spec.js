import React from 'react'
import { shallow } from 'enzyme'
import Header  from './Header'

describe('Header', () => {
    it('renders <header>', () => {
        const header = shallow(<Header />)
        expect(header.find('header')).toHaveLength(1)
    })
})