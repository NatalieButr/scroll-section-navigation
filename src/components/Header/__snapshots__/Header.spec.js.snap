// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders <header> 1`] = `"<header><div class=\\"header_logo\\"><p class=\\"text header_logo__text\\">Harry Potter Section Navigation</p></div></header>"`;
